inc/namespace.php

```diff
@@ -161,8 +161,14 @@ function render_block_search( string $block_content, array $block, \WP_Block $in
 
 	$query_var = sprintf( 'query-%d-s', $instance->context['queryId'] ?? 0 );
 
+	$value = wp_unslash( $_GET[ $query_var ] ?? '' );
+	$value = urldecode( $value );
+	$value = wp_check_invalid_utf8( $value );
+	$value = wp_pre_kses_less_than( $value );
+	$value = strip_tags( $value );
+
```

Comment:
I think its worth adding an inline comment here just to remind us why it was done in this way. 

```suggestion
	// Note sanitize_text_field trims whitespace from start/end of string causing unexpected behaviour.
	$value = wp_unslash( $_GET[ $query_var ] ?? '' );
	$value = urldecode( $value );
	$value = wp_check_invalid_utf8( $value );
	$value = wp_pre_kses_less_than( $value );
	$value = strip_tags( $value );

```